### ISSUE TITLE
Should lighthouse support desktop sites?

### ISSUE THREAD

**Issue by jessiehan** [GoogleChrome/lighthouse#4769]
And the test environment should be different from mobile, maybe lower latency, faster speed. Otherwise, the fmp score of desktop sites would be lower than mobile sites, cause generally, the size of desktop pages is larger.
Do u have any plan or suggestion about that?

**Comment by patrickhulce** [comment]
@jessiehan yes! from the CLI you can control throttling and form factor with flags `--disable-device-emulation --disable-cpu-throttling --disable-network-throttling` and dropdowns to control the same settings are available in Chrome Canary's DevTools. Feel free to try it out and give us feedback if things aren't working out!

**Comment by jessiehan** [comment]
Thank u for ur kindly reply!
As for mobile, Lighthouse uses 3G network, so for desktop, do u have any suggestion about the testing network？
If Lighthouse provide a standard, we can test desktop pages at the same environment, then the score could be more meaningful and comparable.

**Comment by patrickhulce** [comment]
Ah, yes! We are working on multiple throttling profile recommendations for v3 that should include more options than our current 3G (also #4594). 

For now, when testing desktop experience our recommendation is to disable network throttling entirely. If you're wanting to use [more advanced throttling methods](https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md#how-do-i-get-high-quality-packet-level-throttling) and are looking for consistent settings, for cable use 5mbps down, 28ms RTT and for DSL use 1.5mbps down, 50ms RTT.
